test/suites/container_devices_nic_bridged_filtering.sh

```diff
@@ -142,6 +142,89 @@ test_container_devices_nic_bridged_filtering() {
       false
   fi
 
+  # Make sure br_netfilter is loaded, needed for IPv6 filtering.
+  modprobe br_netfilter
```

Comment:
This needs a `|| true` or it will cause test failures when br_netfilter isn't available.

Reply:
updated